src/plugins/kibana/public/settings/styles/main.less

```diff
@@ -211,3 +211,18 @@ kbn-settings-indices {
     }
   }
 }
+
+.wizard-step-title {
```

Comment:
not worried about this yet, but eventually it may make these styles easier to edit if they're split into their own file